astroquery/jplhorizons/core.py

```diff
@@ -149,6 +149,7 @@ def ephemerides_async(self, airmass_lessthan=99,
                           refsystem='J2000',
                           closest_apparition=False, no_fragments=False,
                           quantities=conf.eph_quantities,
+                          rts_flag=None,
```

Comment:
this breaks API. Either add to the end of kwargs, or make kwargs keyword only by inserting a `*` after the mandatory parameters.